### PR TITLE
Html5-6871 build fix post migration to angular9

### PR DESCRIPTION
**Pull request by skasgarv55**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -34,6 +34,7 @@
     "@types/jasmine": "~2.8.3",
     "@types/jasminewd2": "~2.0.2",
     "@types/node": "^12.11.1",
+    "@int/geotoolkit": "^3.0.56",
```
Comment (lukdmitriy) [inline package.json:37]: I think you need to add "@int/impl": "^3.0.56", too before

Comment (skasgarv55) [reply package.json:37]: It was not used any where. When I tested, the local build passed and the application loaded without any console errors. Hence did not add.

But if there is any dependency or any case I may have not tested, then I will add it.

Your thoughts?

Comment (lukdmitriy) [reply package.json:37]: yes, geotoolkit depends on it

Comment (skasgarv55) [reply package.json:37]: Fixed.

**Pull request closed without merging**